node/task.ts

```diff
@@ -14,6 +14,7 @@ require('./extensions');
 
 export enum TaskResult {
     Succeeded = 0,
+    SucceededWithIssues = 2,
```

Comment:
Agreed, but I'm wondering what it will do to tasks that use structures like this:

https://github.com/Microsoft/vsts-tasks/blob/f3bcf5ff1382a435505df1f5b76851ecc850f671/Tasks/Npm/npmtask.ts#L37

It's why I'm not sure what value to give here to be honest...